src/opnsense/scripts/suricata/metadata/rules/et-open.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<ruleset documentation_url="https://doc.emergingthreats.net/bin/view/Main/EmergingFAQ">
+<ruleset documentation_url="https://rules.emergingthreatspro.com/">
```

Comment:
the new url should be https://community.emergingthreats.net/t/frequently-asked-questions/56 if I'm not mistaken.

Reply:
Thank you, this URL looks way better than mine.